### PR TITLE
Merge INSTALL documentation into README

### PR DESCRIPTION
**Pull request by Martchus**
* Remove parts which were redundant with README
* Update outdated parts
* Refer to the openQA documentation for openSUSE repository setup
* Merge remaining parts into README
* See https://progress.opensuse.org/issues/68684

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -95,8 +87,16 @@ requests for consideration or create an issue with a code change proposal.
...
-The required dependencies are delcared in `dependencies.yaml`. (The names listed within
-that file are specific to openSUSE but can be easily transferred to other distributions.)
+Installing dependencies
+^^^^^^^^^^^^^^^^^^^^^^^
+
+On openSUSE one can install the package
+link:https://build.opensuse.org/project/show/devel:openQA[`os-autoinst-devel`] which
+pulls all dependencies required to build with CMake.
+
+The required dependencies are also delcared in `dependencies.yaml`. (The names listed
```
Comment (okurz) [inline README.asciidoc:97]: "declared"

File: README.asciidoc [hunk 2]
```diff
@@ -201,3 +201,62 @@ Run individual tests by specifying them explicitly:
 ----
 make check TESTS=23-baseclass.t
 ----
+
+How to run test cases
+---------------------
+
+This following instructions shows how to run test cases using the openSUSE test
```
Comment (okurz) [inline README.asciidoc:208]: I would prefer if you describe the much simpler os-autoinst-distri-example but I don't mind if we mention the openSUSE tests as a real-life example :)

File: README.asciidoc [hunk 3]
```diff
@@ -201,3 +201,62 @@ Run individual tests by specifying them explicitly:
...
+distribution as an example.
+
+Clone the test distribution and needles:
+
+-----------------------------------------------------------------------------
+mkdir distri
+cd distri
+git clone git@github.com:os-autoinst/os-autoinst-distri-opensuse.git opensuse
+cd opensuse
+git clone git@github.com:os-autoinst/os-autoinst-needles-opensuse.git needles
```
Comment (okurz) [inline README.asciidoc:218]: That will actually not work as needles would need to be cloned into the products subdir but please follow my above suggestion

File: README.asciidoc (hunk 2 above)
Comment (Martchus) [reply README.asciidoc]: I added both examples.

Comment (okurz) [reply README.asciidoc]: Well, thanks for the effort but you misread. I meant in particular https://github.com/os-autoinst/os-autoinst-distri-example , not the openQA-in-openQA tests. The main reason being that https://github.com/os-autoinst/os-autoinst-distri-example has no separate needle git repo which is a test distribution specific setup that is not necessary in general.

Comment (Martchus) [reply README.asciidoc]: I've included a link to the minimum example as well.

**Pull request merged**